CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.1.1](https://github.com/sonata-project/SonataPageBundle/compare/3.1.0...3.1.1) - 2016-08-01
```

Comment:
He was talking about [this](https://github.com/sonata-project/SonataAdminBundle/pull/4102#issue-173452628) (in the PR body, not in the file)